### DISCUSSION TITLE
[Front End] Vue3 + TypeScript

### DISCUSSION THREAD

**Discussion by rofixro** [realworld-apps/realworld#1375] [Added implementations]
Hello everyone, I have published a [vue3-realworld-app](https://github.com/niyaco/vue3-realworld-app) front-end project built with Vue3. I know there are projects written with Vue3 already, but I'd like to provide some different coding approaches to help partners who are trying web development for the first time. Finally, thank you very much to the authors and maintainers who open sourced this project. Thank you all.

**Comment by geromegrignon** [comment]
Hello, implementation added to the listing. That's great to see an implementation using Pinia!
